src/dataStructures.ts

```diff
@@ -0,0 +1,74 @@
+/* -----------------------------------------------------------------------
+                    Virtual Field Environment Software
+   -----------------------------------------------------------------------
+    Instantiation of the objects that will be used for interacting with 
+    Photo Sphere Viewer to build a virtual geology field guide.
+   ----------------------------------------------------------------------- */
+
+// Virtual Field Environment: the total collection of photo sphere environments
+export interface VFE {
+  name: string;
+  map: Map<string, number>; // arguments <"imgURL", center_number>
+  photospere: photosphere[];
+}
+
+// Navigation map: a birdseye view of the various hotspots within a single 360-environment
+export interface navMap {
+  src: string;
+  hotspot: hotspotData[];
+}
+
+// Photosphere: a single 360-environment
+export interface photosphere {
+  id: string;
+  src: string;
+  hotspot: hotspotData[];
```

Comment:
The hotspot data type doesn't contain any information about the hotspot's position.
It should be an array of 3D hotspots instead.